src/lib/array.jl

```diff
@@ -41,6 +41,15 @@ end
   return (_project(x, dx), map(_->nothing, inds)...)
 end
 
+using CUDA
+import NNlib
+
+∇getindex(x::CuArray, inds::Tuple{AbstractArray{<:Integer}}) = dy -> begin
+  dx = _zero(x, eltype(dy))
```

Comment:
we have a `_zero` function ?

Reply:
Yes, this line is copied from the existing function above.